src/classes/groundplaces.ts

```diff
@@ -0,0 +1,131 @@
+
+import { GroundPlace, StopGroup, StopCluster, Properties } from '../types';
+
+interface Filters {}
+
+interface GroundPlacesDiff {}
+
+/**
+ * @description GroundPlaces business logic.
+ */
+export class GroundPlaces {
+
+  /**
+   * @description Returns a list of places.
+   * @param query Can be a name, a Gpuid, a unique name or other name.
+   * @param filters Filters with different options (StopGroup, StopCluster, Serviced, SegmentProvider).
+   * @returns {Array<StopGroup|StopCluster>}
+   */
+  // @ts-ignore
+  public autocomplete(query: string, filters: Filters): (StopGroup | StopCluster)[] {}
+
+  /**
+   * @description Create stopCluster or stopGroup.
+   * @param type Type can be stopCluster or stopGroup.
+   * @param groundPlace Informations to use for creating the place.
+   * @param GpuidParent Ground place unique identifier of the stopCluster parent if the type is a stopGroup. 
+   * @returns {StopGroup|StopCluster}
+   */
+  // @ts-ignore
+  // This method will call Storage.createStopGroup or Storage.createStopCluster depending on type passed in params.
+  public create(type: 'stopCluster' | 'stopGroup', groundPlace: GroundPlace, GpuidParent?: string): StopGroup | StopCluster {}
```

Comment:
Je séparerais les deux méthodes de façon explicite. 
Sans les refactoriser dans un "create" mix

Reply:
I agree with @Simobain , you should create custom types to reuse them.

Reply:
Done